cmd.go

```diff
@@ -167,19 +168,29 @@ func abtestCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			parsedSeed := parseAndSetSeed(seed)
 
-			conn1 := setUpDatabaseConnection(dsn1)
-			conn2 := setUpDatabaseConnection(dsn2)
+			var conn1, conn2 *sql.Conn
+			if !debug {
+				conn1 = setUpDatabaseConnection(dsn1)
+				conn2 = setUpDatabaseConnection(dsn2)
+			}
```

Comment:
This option sounds like "dryRun" instead of "debug".

Reply:
reverted, this PR should focus on generating json related data